### PR TITLE
fix example code in celoss (complement pr 56207)

### PR DESCRIPTION
**Pull request by MqLeet**
### PR types
Others

### PR changes
Docs

### Description
Fix the example code in both nn.layer.CrossEntropyLoss and nn.functional.cross_entropy, this is the supplementary of PR https://github.com/PaddlePaddle/Paddle/pull/56207#pullrequestreview-1619441072 
1. add `>>>` into annotation line
2. add integer label & one-hot label into example code, demonstrate the usage of `label_smoothing`

### CODE REVIEW

**Review by zhwesky2010 — APPROVED** [review]
LGTM

**Review by sunzhongkai588 — APPROVED** [review]
LGTM for docs

**Review by tianshuo78520a — APPROVED** [review]
